plugins/scaffolder/src/components/TemplateEditorPage/DryRunResults/DryRunResultsView.tsx

```diff
@@ -24,11 +24,13 @@ import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import CodeMirror from '@uiw/react-codemirror';
 import React, { useEffect, useMemo, useState } from 'react';
-import { TaskStatusStepper } from '../../TaskPage/TaskPage';
-import { TaskPageLinks } from '../../TaskPage/TaskPageLinks';
+
 import { useDryRun } from '../DryRunContext';
-import { FileBrowser } from '../../FileBrowser';
+
 import { DryRunResultsSplitView } from './DryRunResultsSplitView';
+import { FileBrowser } from '../../../components/FileBrowser';
+import { TaskStatusStepper } from '../../../legacy/TaskPage/TaskPage';
+import { TaskPageLinks } from '../../../legacy/TaskPage/TaskPageLinks';
```

Comment:
Would be good to flip this around right? trying to get to a state where we can simply delete the  `legacy` folder once it's ready.

Reply:
yeah indeed. that was one of my thoughts.